assets/javascripts/views/layout/document.coffee

```diff
@@ -46,6 +46,27 @@ class app.views.Document extends app.View
     app.appCache?.updateInBackground()
     return
 
+  hideSidebar: (saveLayout = true) ->
```

Comment:
I'll see how I can do this without making the `toggleSidebar` function too messy